program/tests/accounts.rs

```diff
@@ -195,8 +180,7 @@ async fn fail_account_checks(test_mode: TestMode, include_onramp: bool) {
 
         // while onramp is optional, an incorrect onramp misaligns all subsequent accounts
         // this is not a problem for the program and causes the mint to fail to validate, but requires tweaking this test
-        // NOTE once deposit/withdraw require onramp, delete this block
-        if include_onramp && instruction_pubkey == accounts.pool {
+        if !remove_onramp && instruction_pubkey == accounts.pool {
```

Comment:
i realized this comment is pointless because once they require onramp we have to get rid of _everything_ involving `remove_onramp`